content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -25,81 +31,74 @@ depends on the stage of the manufacturing process.
    :guilabel:`Search...` bar to display all virtual locations. From the list, select the
    :guilabel:`Virtual Locations/Scrap` location.
 
-Scrap manufacturing components
-==============================
+.. _scrap_components/scrap-window:
 
-To scrap components during the manufacturing process, begin by navigating to
-:menuselection:`Manufacturing --> Manufacturing Orders`, then select a manufacturing order or click
-:guilabel:`Create` to configure a new one. If a new manufacturing order is created, select a product
-from the :guilabel:`Product` drop-down menu, then click :guilabel:`Confirm`.
+Scrap pop-up window
+===================
 
-Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
-the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+Scrapping components and finished products is done through the :guilabel:`Scrap` pop-up window.
+The pop-up window can be accessed from an |MO| in the backend, or the Shop Floor module.
 
-.. image:: scrap_manufacturing/scrap-button.png
-   :align: center
-   :alt: The scrap button on a manufacturing order.
+Scrap component from Manufacturing
+----------------------------------
 
-From the :guilabel:`Product` drop-down menu on the :guilabel:`Scrap` pop-up window, select the
-component that is being scrapped, then enter the quantity in the :guilabel:`Quantity` field.
-Finally, click :guilabel:`Done` to scrap the component.
+To scrap a component from an |MO|, begin by navigating to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`, and then select an |MO|. At the top of the |MO|, click the
+:guilabel:`Scrap` button to open the :guilabel:`Scrap` pop-up window.
 
-.. image:: scrap_manufacturing/scrap-pop-up.png
-   :align: center
-   :alt: The Scrap pop-up window.
+Scrap finished product from Manufacturing
+-----------------------------------------
 
-.. important::
-   Before clicking :guilabel:`Mark As Done` on a manufacturing order, only the components of the
-   finished product can be scrapped, **not** the finished product itself. This is because Odoo
-   recognizes that the finished product cannot be scrapped before it has been manufactured.
+To scrap a finished product from an |MO|, begin by navigating to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`. Select an open |MO|, and then click the :guilabel:`Produce
+All` button to close it.
 
-After scrapping a component, continue the manufacturing process using the required quantity of the
-component that was scrapped. The on-hand stock count for the component that was scrapped updates to
-reflect both the scrapped quantity and the quantity consumed during manufacturing.
+To select an |MO| that has already been closed, navigate to :menuselection:`Manufacturing -->
+Operations --> Manufacturing Orders`, remove the :guilabel:`To Do` filter from the
+:guilabel:`Search...` bar, and then select the desired |MO|.
```

Comment:
same search bar comment as above... let's go with :guilabel:`Search` bar, per the decision you got from ZST the other day (just removing the "...")

Reply:
Same comment as above.